sim/pokemon.js

```diff
@@ -277,6 +277,11 @@ class Pokemon {
 		this.boosts = {atk: 0, def: 0, spa: 0, spd: 0, spe: 0, accuracy: 0, evasion: 0};
 		/**@type {{[k: string]: number}} */
 		this.stats = {atk: 0, def: 0, spa: 0, spd: 0, spe: 0};
+		/**
+		 * Commented out until a use is found for Combat Power in Let's Go
+		 * @type {number?}
+		 * this.combatPower = null;
```

Comment:
This definitely doesn't need to be here at all. Just recalculate Combat Power every time it's needed.